mico-core/src/main/java/io/github/ust/mico/core/model/MicoApplicationDeploymentInfo.java

```diff
@@ -41,6 +43,7 @@
 @AllArgsConstructor
 @Accessors(chain = true)
 @NodeEntity
+@Deprecated
```

Comment:
Why is there deprecated? Do we have a replacement for it?

Reply:
Jakob is currently working on a better solution for it. See issue #288 for it.